dev/build/windows/makecoq_mingw.sh

```diff
@@ -1904,6 +1955,34 @@ function make_addon_gappa {
   fi
 }
 
+# elpi
```

Comment:
It would be nice to a short description what Elpi is.

Reply:
wiil do